qa/tasks/cephfs/test_admin.py

```diff
@@ -194,6 +195,58 @@ def test_fs_new_pool_application_metadata(self):
             self._check_pool_application_metadata_key_value(
                 pool_names[i], 'cephfs', keys[i], fs_name)
 
+    def test_fs_rename(self):
+        """
+        That the file system can be renamed, and the application metadata set on its pools are as expected.
+        """
+        self.mount_a.umount_wait(require_clean=True)
```

Comment:
Does renaming a fs also happen to break this mount?

Reply:
let me check up on this.

Reply:
If the mount client uses fs specific auth ID, then after `fs rename` you can no longer read/write from the mount. Even if you update the caps of the auth ID to the new file system name, you can't access the file system unless you umount and re-mount.

Reply:
Please leave a comment for this.

Reply:
Done